website/content/docs/intro/use-cases/dynamic-config-vault-dynamic-secrets.mdx

```diff
@@ -0,0 +1,269 @@
+---
+layout: docs
+page_title: Dynamic App Config using Vault Dynamic Secrets
+description: |-
+  Waypoint enables you to source config dynamically from various external data sources using config sourcer plugins - this use case depicts doing just that with Vault dynamic database secrets.
+---
+
+# Dynamic Secrets for Waypoint with Vault
+
+Hashicorp Waypoint is an application deployment platform that simplifies how users
+test and release apps onto platforms and utilize infrastructure managed by Terraform.
```

Comment:
Do we need this preamble, or can we assume readers will already have a basic grasp of what Waypoint does?

Comparing with [Vault’s use-case tutorials](https://developer.hashicorp.com/vault/tutorials/db-credentials/database-secrets), it seems like they dive straight into talking about the feature in question, rather than setting the stage with a general pitch for the product.

Reply:
I think this can be removed! I included it at the beginning when I was drafting this as a blog post for a quick intro, but since this is embedded in our website I think we can assume the reader went through the introduction first. 😄